clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -657,6 +657,11 @@ private void checkDescribeConfigsResponseVersions() {
     private void checkErrorResponse(AbstractRequest req, Throwable e, boolean checkEqualityAndHashCode) {
         AbstractResponse response = req.getErrorResponse(e);
         checkResponse(response, req.version(), checkEqualityAndHashCode);
+        Errors error = Errors.forException(e);
+        Map<Errors, Integer> errorCounts = response.errorCounts();
+        assertEquals(Collections.singleton(error), errorCounts.keySet(),
+            "API Key " + req.apiKey().name + " v" + req.version() + " failed errorCounts test");
+        assertTrue(errorCounts.get(error) > 0);
```

Comment:
Do we need to get this change as well: https://github.com/apache/kafka/pull/11288/files#diff-b8c878a734acf31ecab2f39b75acc71f6a6f9b4cdad66c18824d19e8f1a11284R2887?

Reply:
It has no impact on functionality. It just makes the request looks better as by defaut the topic name is empty string. Happy to add it if you prefer having it.

Reply:
Understood, thanks.